src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/ContentHelper.Common.cs

```diff
@@ -54,9 +51,7 @@ internal static StringBuilder GetRawContentHeader(HttpResponseMessage response)
                     // Headers may have multiple entries with different values
                     foreach (var headerValue in header.Value)
                     {
-                        raw.Append(header.Key);
-                        raw.Append(": ");
-                        raw.Append(headerValue);
+                        raw.Append($"{header.Key}: {headerValue}");
                         raw.AppendLine();
```

Comment:
We can combine all in one.
`raw.AppendLine($"{header.Key}: {headerValue}");`

Reply:
done